src/Context/WishlistContext.jsx

```diff
@@ -22,15 +22,13 @@ const WishlistProvider =({children})=>{
     const [state,dispatchWishlist]=useReducer((state,action)=>{
     switch (action.type){
         case "ADD_TO_WISHLIST":
-            return {...state, wishlistItems : state.wishlistItems + 1, price: state.price+action.payload.price,
-                totalPrice : Number(state.totalPrice) + Number(action.payload.price), wishlistList: addToWishlistList(state,action)};
+            return {...state, wishlistItems : state.wishlistItems + 1, wishlistList: addToWishlistList(state,action)};
```

Comment:
Remove the **wishlistItems variable** here. 

Since you have not added in your initial State it would give you *error*.